odoo/addons/base/models/res_currency.py

```diff
@@ -313,11 +313,15 @@ def _get_view(self, view_id=None, view_type='form', **options):
         arch, view = super()._get_view(view_id, view_type, **options)
         if view_type in ('tree', 'form'):
             currency_name = (self.env['res.company'].browse(self._context.get('company_id')) or self.env.company.root_id).currency_id.name
-            for field in [['company_rate', _('Unit per %s', currency_name)],
-                          ['inverse_company_rate', _('%s per Unit', currency_name)]]:
-                node = arch.xpath("//tree//field[@name='%s']" % field[0])
+            fields_maps = [
+                [['company_rate', 'inverse_rate'], _('Unit per %s', currency_name)],
+                [['inverse_company_rate', 'rate'], _('%s per Unit', currency_name)],
+            ]
+            for fields_map in fields_maps:
+                xpath_expression = '//tree//field[' + " or ".join([f"@name='{f}'" for f in fields_map[0]]) + "][1]"
```

Comment:
nitpick: don't create lists where not needed
```suggestion
                xpath_expression = '//tree//field[' + " or ".join(f"@name='{f}'" for f in fields_map[0]) + "][1]"
```